source/index.html.md

```diff
@@ -791,7 +791,337 @@ Parameter | Description
+                    "has_children": false,
+                    "name": "Product Revenue",
+                    "active": true,
+                    "write_protection": "NONE",
+                    "special_case_id": null,
+                    "date": null
+                }
+            ],
+            "Scenario": [
+                {
+                    "id": 27184,
+                    "formula": null,
+                    "has_children": false,
+                    "name": "Actuals",
+                    "active": true,
+                    "write_protection": "ACTUALS",
+                    "special_case_id": null,
+                    "date": null
+                }
+            ],
+            "Time": [
+                {
+                    "id": 27205,
+                    "formula": null,
+                    "has_children": true,
+                    "name": "2019",
+                    "active": true,
+                    "write_protection": "NONE",
+                    "special_case_id": null,
+                    "date": "2019-12-31"
+                },
+                {
+                    "id": 27206,
+                    "formula": null,
+                    "has_children": true,
+                    "name": "Q1-19",
+                    "active": true,
+                    "write_protection": "NONE",
+                    "special_case_id": null,
+                    "date": "2019-03-31"
+                },
+                {
+                    "id": 27207,
+                    "formula": null,
+                    "has_children": false,
+                    "name": "Jan-19",
+                    "active": true,
+                    "write_protection": "NONE",
+                    "special_case_id": null,
+                    "date": "2019-01-31"
+                }
+            ],
+            "Department": [
+                {
+                    "id": 27182,
+                    "formula": null,
+                    "has_children": false,
+                    "name": "Department - Other",
+                    "active": true,
+                    "write_protection": "NONE",
+                    "special_case_id": "DEFAULT",
+                    "date": null
+                }
+            ]
+        },
+        "attributes": {},
+        "splits": []
+    }
+]
+```
+
+This endpoint retrieves any values in the cube that make up the overall value at the provided cross section of dimensions. For instance, in the example JSON "Q1-19" was provided as the "Time" dimension and you can see that it returned a value for "Jan-19", "Feb-19", and "Mar-19" which all make up "Q1-19".
+
+Drilling down in to a cube value is very similar to [retrieving data by slices](#get-data-by-slice) in that you are able to pass either the names of the dimensions or their IDs as parameters. A dimension value must be provided for each top level dimension then this endpoint will simply return any numbers that go in to that number.
```

Comment:
> then this endpoint will simply return any numbers that go in to that number.

You could delete this. You already said this more clearly in the first paragraph, and it sounds kind of weird to me:  "numbers that __go into__ a number"